File: day2/loui/code.jl

open("input.txt", "r") do file
    lines = readlines(file)
    data = [parse.(Int64, split(line)) for line in lines]
    # println(data)

    assending_matrix = [row for row in data if row == sort(row)]
    descending_matrix = [row for row in data if row == sort(row, rev=true)]
    
    # matrix that includes only truly assending and descending rows of the data set
    combined_matrix = vcat(assending_matrix, descending_matrix)
    
    
end

File: day2/loui/code.py
import numpy as np

# Function to read the content of input.txt and convert it to a list of lists
def read_input_file(filename):
    with open(filename, 'r') as file:
        lines = file.readlines()
        data = [list(map(int, line.split())) for line in lines]
    return data

data = np.array(read_input_file('input.txt'))

